KsApi/models/graphql/adapters/Reward+RewardFragment.swift

```diff
@@ -26,14 +26,16 @@ extension Reward {
     let estimatedDeliveryOn = rewardFragment.estimatedDeliveryOn
       .flatMap(dateFormatter.date(from:))?.timeIntervalSince1970
 
+    let rewardHasNoAddons = rewardFragment.allowedAddons.nodes?.isEmpty ?? true
```

Comment:
I think this would read better as an argument below if it was `rewardHasAddOns` (invert logic) but that is just a personal style preference.